src/localization/relocalizers/lsh_cv_hashing.h

```diff
@@ -27,6 +27,7 @@
 #include "database/online_database.h"
 #include "features/ifeature.h"
 #include "relocalizers/irelocalizer.h"
+#include <memory>
```

Comment:
nit: It seems to be a nice style to separate the local includes from the system ones by 1 empty line.

Reply:
done.